python/postprocessing/modules/btv/btagSFProducer.py

```diff
@@ -227,6 +227,12 @@ def getFlavorBTV(self, flavor):
         return flavor_btv
 
     def getSFs(self, jet_data, syst, reader,  measurement_type = 'auto', shape_corr = False):
+        if reader is None:
+            if self.verbose > 0
```

Comment:
Two problems here: 1) a colon missing missing after `self.verbose > 0`; 2) the SFs are now -1 if the corrections are unavailable (b/c you bypass `sf < 0.01` check in this function), but they should be 1. You can view my fixes here: https://github.com/HEP-KBFI/nanoAOD-tools/commit/692d41170d95ef0c622d180653de455d0f801484

Reply:
Yeah. Unfortunately, that part was done in a hurry. Thanks for checking.

@veelken @rappoccio Are there any plans to add unit tests to nanoAOD-tools?
Since there is a standalone mode, it would be possible to add travic-ci or gilab-ci (like for the CMSSW NanoAOD repo), would it not? I would be happy to write some unit test for this PR if the structure is clear.